### PR TITLE
BZ#1702434 - updated unmanaged bootloader files

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* Nightly

Packaging note: This has been merged downstream by accident. This
repository used to be downstream only but since we no longer ship any
binaries in here, it's upstream now too:

commit 6b58570dfb34b64071a7277ca7086042db6cae04
Author: Lukas Zapletal <lzap+git@redhat.com>
Date:   Fri Apr 17 16:07:46 2020 +0200
BZ#1702434 - updated unmanaged files

---

### CODE REVIEW

File: packages/foreman/foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Name: foreman-bootloaders-redhat
 Version: 201901011200
```
Comment (evgeni) [inline packages/foreman/foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec:2]:
```suggestion
Version: 202004171200
```

;-)

Comment (lzap) [reply packages/foreman/foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec]: Done

**Comment by lzap** [comment]
```
[2020-05-20T12:42:01.530Z]     Added downloaded_rpms repo from ./downloaded_rpms/el8
[2020-05-20T12:42:01.530Z]     package: rubygem-hammer_cli-2.2.0-0.1.pre.develop.20200518090133git919dc54.el8.noarch from el8-foreman-nightly
[2020-05-20T12:42:01.530Z]       unresolved deps:
[2020-05-20T12:42:01.530Z]         rubygem(amazing_print)
```

**Comment by evgeni** [comment]
repoclosure unrelated

**Pull request merged**